docs/source/security/cors.mdx

```diff
@@ -84,108 +82,135 @@ If you create a public API or an API to embed in websites you don't control your
+    // highlight-start
+    cors<cors.CorsRequest>(corsOptions),
+    // highlight-end
+    json(),
+    expressMiddleware(server),
+  );
+
+  await new Promise<void>((resolve) => httpServer.listen({ port: 4000 }, resolve));
+  console.log(`🚀 Server ready at http://localhost:4000/graphql`);
+}
 ```
 
-Under the hood, the batteries-included `apollo-server` wraps around middleware provided by the [`cors`](https://github.com/expressjs/cors#configuration-options) npm package, enabling you to configure the `Access-Control-Allow-Origin` header via the `origin` option. The example above enables CORS requests from `https://www.your-app.example`, along with `https://studio.apollographql.com`.
+</MultiCodeBlock>
 
-> Note if you plan to use [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/explorer/) as a GraphQL web IDE you should include `https://studio.apollographql.com` in your list of valid origins. However, if you choose to embed the [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/embed-explorer/) or [Sandbox](https://www.apollographql.com/docs/studio/explorer/sandbox/#embedding-sandbox), you don't need to add Studio to your CORS configuration at all. Requests will go through the page embedding Studio.
+Using the `cors` package directly, we can configure the `Access-Control-Allow-Origin` header using the [`origin` option](https://github.com/expressjs/cors#configuration-options). The example above enables CORS requests from `https://www.your-app.example`, along with `https://studio.apollographql.com`.
 
-If you're using another integration of Apollo Server, you can add the `cors` configuration option to your framework-specific middleware function to change your server's CORS behavior. To learn more about the CORS defaults and options for your integration of Apollo Server, see [CORS configuration options](../api/apollo-server/#cors-1).
+> Note: if you plan to use [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/explorer/) as a GraphQL web IDE, you should include `https://studio.apollographql.com` in your list of valid origins. However, if you plan to embed the [Explorer](https://www.apollographql.com/docs/studio/explorer/embed-explorer/) or use [Apollo Sandbox](https://www.apollographql.com/docs/studio/explorer/sandbox), you *don't* need to specify Studio's URL in your CORS origins because requests will go through the page embedding Studio.
 
-Below is an example of setting up CORS with the [`apollo-server-express`](../integrations/middleware#swapping-out-apollo-server) package:
+If you're using another integration of Apollo Server, you can add and configure CORS for your server using your framework's standard functionality. 
 
-```js
-// ... set up the server
-await server.start();
+You can also choose to remove CORS middleware entirely to disable cross-origin requests. If you are using `expressMiddleware`, you can do this by passing `false` to the `cors` function:
```

Comment:
Nah, you just don't use `cors` at all in this case.

Reply:
Oh right, that is a way better way to do it 😅

So this becomes:

> You can also choose to omit CORS middleware entirely to disable cross-origin requests. This is [recommended for subgraphs in a federated graph](/federation/subgraphs/#securing-your-subgraphs).